src/pages/login/Login.js

```diff
@@ -0,0 +1,94 @@
+import React, {useState} from 'react';
+import {useNavigate} from "react-router-dom";
+import './Login.css';
+import user from "../../data/user.json";
+
+
+const Login = ({toggleIsAuthenticated}) => {
+
+    const [formState, setFormState] = useState({
+        email: "",
+        password: "",
+
+    });
+
+    const navigate = useNavigate()
+
+    function handleLogin() {
+
+        const field = user.find((log) => {
+            return log.emailadres === formState.email
+        })
+        console.log(field)
+
+        if (field.emailadres === formState.email && field.password === formState.password) {
+            toggleIsAuthenticated(true);
+            navigate("/blogposts")
+        }
+        // else if(field.emailadres !== formState.email && field.password !== formState.password){
+        //     toggleIsAuthenticated(false);
+        //
+        // }
+    }
+
+
+    function handleChange(event) {
+        const changedFieldName = event.target.name;
+        const newValue = event.target.type === "checkbox" ? event.target.checked : event.target.value;
```

Comment:
"checkbox" ? event.target.checked : kan we, je hebt geen checkbox in je login zitten